clients/client-python/gravitino/filesystem/gvfs.py

```diff
@@ -590,7 +606,8 @@ def _convert_actual_info(
             "name": path,
             "size": entry["size"],
             "type": entry["type"],
-            "mtime": entry["mtime"],
+            # Some file systems may not support the `mtime` field.
```

Comment:
It's better to specify which file systems are not supported.

Reply:
Okay